spec/jobs/resolution_proofing_job_spec.rb

```diff
@@ -107,6 +110,13 @@
 
         allow(state_id_proofer).to receive(:proof).
           and_return(Proofing::Result.new(transaction_id: aamva_transaction_id))
+
+        ServiceProvider.create(
+          issuer: issuer,
+          friendly_name: issuer,
+          app_id: issuer,
```

Comment:
Let's please use different values for issuer, friendly_name, app_id. They're used in different places and this way we can make sure we're reading from the right place, doing this could mask bugs